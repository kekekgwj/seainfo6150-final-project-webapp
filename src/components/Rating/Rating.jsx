import React from "react";
import './Rating.css';

const Rating = ( {getRatingScore} ) => {

    return (
        <div className="rate-container">
            <div className="rate-content" onClick={getRatingScore}>
                <input type="radio" name="rate" value="1"/>
                <input type="radio" name="rate" value="2"/>
                <input type="radio" name="rate" value="3"/>
                <input type="radio" name="rate" value="4"/>
                <input type="radio" name="rate" value="5"/>
            </div>
        </div>
    )
}

export default Rating;