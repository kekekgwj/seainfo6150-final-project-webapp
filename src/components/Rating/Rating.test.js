import React from "react";
import Rating from "./Rating";


describe("rating  tests", () => {
  it("renders correctly", () => {
    const { container } = render(<Rating getRatingScore={()=>{console.log('get rating score')}}/>);
    expect(container).toMatchSnapshot();
  });
});