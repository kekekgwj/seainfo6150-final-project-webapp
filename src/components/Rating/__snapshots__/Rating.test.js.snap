// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rating  tests renders correctly 1`] = `
<div>
  <div
    class="rate-container"
  >
    <div
      class="rate-content"
    >
      <input
        name="rate"
        type="radio"
        value="1"
      />
      <input
        name="rate"
        type="radio"
        value="2"
      />
      <input
        name="rate"
        type="radio"
        value="3"
      />
      <input
        name="rate"
        type="radio"
        value="4"
      />
      <input
        name="rate"
        type="radio"
        value="5"
      />
    </div>
  </div>
</div>
`;
